sim/core/database.go

```diff
@@ -467,6 +719,20 @@ func (equipment *Equipment) Stats() stats.Stats {
 	return equipStats
 }
 
+func (e *Item) RandPropPoints(itemLevel int32) int32 {
```

Comment:
Do any of the spec unit test configs still have random suffix items in them? If not, it's important to add at least one gear config where this code would be run during testing.